rs/bitcoin/adapter/tests/adapter_test.rs

```diff
@@ -183,12 +185,32 @@ fn start_adapter_and_client(
     let anchor: BlockHash = "0000000000000000035908aacac4c97fb4e172a1758bbbba2ee2b188765780eb"
         .parse()
         .unwrap();
-    // We send this request to make sure the adapter is not idle.
-    let _ = make_get_successors_request(&res.0, anchor.to_vec(), vec![]);
+    if !is_idle {
```

Comment:
If you go with the suggestion above you can change this to
```suggestion
    if let AdapterState::Idle = adapter_state {
```

Reply:
thanks